test/hack/e2e_scripts/install_karpenter.sh

```diff
@@ -1,7 +1,13 @@
 aws eks update-kubeconfig --name "$CLUSTER_NAME"
 
-# Parse minor version to determine whether to enable the webhooks
-K8S_VERSION_MINOR="${K8S_VERSION#*.}"
+# First, conditionally install the webhook stanza and CRDs
+if (( "$WEBHOOKS_ENABLED" == 'false' )); then
+helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd \
+  --namespace kube-system \
+  --version "0.37.5" \
```

Comment:
This is pinning to a version that may change in the future with future patch releases (unless you were just doing this for testing)